### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#69122]
#### VFS team name
Benefit Tools

#### VFS product name
Claim Status Tool

#### VFS feature name
Improved Evidence Request Initiative

#### Point of Contact/Reviewers
Erin White - @erinrwhite - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
The Need Help? section doesn't have all of the information (i.e. TTY) contained in the Need Help? design component.

### Link, screenshot or steps to recreate

<img width="667" alt="Screenshot 2023-11-02 at 2 12 23 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/755248/4f9a2ab8-e9fe-4fa1-8d1e-9ae61416eaed">

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Please incorporate the [need help design component](https://design.va.gov/components/form/need-help) if possible, or update the information in this section to include TTY information.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by skylerschain** [comment]
@erinrwhite @allison0034 @humancompanion-usds 

Thanks for flagging this. Do you all know off the top of your head if there is an instance of the "Need help" component that _is not_ a footer? The [component usage guidelines](https://design.va.gov/components/form/need-help#usage) say to use it "As a footer for form pages in an application." But that is not how it's been used in CST historically – in the CST detail page it floats to the right of the status content (on desktop). 

<img width="1091" alt="Screenshot 2023-11-08 at 9 49 48 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/fedba371-1162-4ffe-bb9c-90ef5b5c8313">

**Comment by jacobworrell** [comment]
@shiragoodman looking closely at this ticket and[ this one](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69124), it seems like our intended fix would solve the issue described in both. The two tickets seem redundant, but I want to make sure I'm not missing an important piece of nuance. Is this your read as well?

**Comment by allison0034** [comment]
@skylerschain if you keep it on the right can you still use the [component](https://design.va.gov/storybook/?path=/docs/components-va-need-help--default)?

**Comment by allison0034** [comment]
@jacobworrell that other ticket is separate since the Need help component does not contain an "Open a live chat" link. Switching to using the Need help component will not fix that link.

**Comment by skylerschain** [comment]
@allison0034 Sorry for the delayed reply. I don't see a compelling reason to break from the usage guidelines here of using it as a footer, so I was thinking of sticking to that placement. It would look something like this.

<img width="939" alt="Screenshot 2023-11-27 at 11 47 44 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/78e3700f-fd09-4ee8-9d94-34afc002e107">
